.travis.yml

```diff
@@ -4,6 +4,19 @@ os: linux
 dist: trusty
 sudo: required
 
+addons:
+  apt:
+    packages:
+      - gcc
+      - gcc-multilib
+      - g++
+      - libc6-dev-i386
+      - g++-multilib
+      - python3-ply
+      - linux-libc-dev
+      - gcc-multilib
+      - build-essential
```

Comment:
Please, move these into a new file under the tools folder (`tools/apt-get-install-zephyr-deps.sh`). Similar to `tools/apt-get-install-qemu-arm.sh`. After that you can add it the the `before_install` step in `travis.yml`.